etl_manager/meta.py

```diff
@@ -175,12 +188,12 @@ def _check_column_does_not_exists(self, column_name) :
         if column_name in self.column_names :
             raise ValueError("The column name provided ({}) already exists table in meta.".format(column_name))
 
-    def update_column(self, column_name, new_name = None, new_type = None, new_description = None) :
+    def update_column(self, column_name, new_name = None, new_type = None, new_description = None, new_pattern = None, new_enum = None, new_nullable = None) :
 
         self._check_column_exists(column_name)
 
-        if new_name is None and new_type is None and new_description is None :
-            raise ValueError("one or more of the function inputs (new_name, new_type and new_description) must be specified.")
+        if new_name is None and new_type is None and new_description is None and new_pattern is None and new_enum is None and new_nullable is None:
```

Comment:
gnarly, probably create an all_none() function that takes a list as an arg?

Reply:
ha! - yes